### ISSUE TITLE
Windows detecting both DS3 and x360 emulated controller (is there a workaround/fix?)

### ISSUE THREAD

**Issue by rjmxtechnology** [ViGEm/DsHidMini#12]
Is there a way to stop windows from detecting the DS3 at the same time as the emulated 360 when using dshidmini with DS4Windows? My games seem to be showing both the xbox and ds3 inputs when I connect a controller

**Comment by AshUchiha** [comment]
Yes tick Hide Ds4 Controller in Ds4Windows settings.

**Comment by nefarius** [comment]
Until there's a better solution available check out the previous comment and the documentation provided in regards to setting up HidGuardian.

**Comment by rjmxtechnology** [comment]
> 
> 
> Yes, tick (enable) Hide Ds4 Controller in Ds4Windows settings.
Thanks! This seemed to do the trick. For some reason I had missed it when looking through the settings and thought it had been removed in an update or something. Guess I was mistaken. 


> 
> 
> Until there's a better solution available check out the previous comment and the documentation provided in regards to setting up HidGuardian.
I did a bit of a readthrough on hidguardian and I was having trouble grasping how exactly to use it and what all the options mean. Is there a good place for documentation on it?
